types/node/v10/util.d.ts

```diff
@@ -87,24 +87,27 @@ declare module "util" {
         fn: (arg1: T1, arg2: T2, arg3: T3, arg4: T4, arg5: T5, arg6: T6) => Promise<TResult>
     ): (arg1: T1, arg2: T2, arg3: T3, arg4: T4, arg5: T5, arg6: T6, callback: (err: NodeJS.ErrnoException | null, result: TResult) => void) => void;
 
+    type Errback = (err?: Error | null) => void;
```

Comment:
Why is this a union type with null?

```typescript
type Errback = (err?: Error) => void;
```

Reply:
I thought that `?` implied `| undefined`, not `| null`, I could be wrong though... 🤔

Reply:
Strictly, speaking, it indicates optionality, but yes, it's `undefined`, not `null`.